NEWS

```diff
@@ -4,6 +4,9 @@ CHANGES WITH 256 in spe:
 
         Announcements of Future Feature Removals and Incompatible Changes:
 
+        * Support for flushing of the nscd cache will be dropped in a future
```

Comment:
maybe be a bit more specific, i.e. say nscd user/group database caches

Reply:
Updated like that.